internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -422,7 +422,7 @@ func LoadConfig(seq runtime.Sequence, data interface{}) (runtime.TaskExecutionFu
 		download := func() error {
 			var b []byte
 
-			b, e := fetchConfig(r)
+			b, e := fetchConfig(ctx, r)
```

Comment:
I don't think that we ever set timeout for each task, so it might make some sense to add 

`ctx, cancel = context.WithTimeout(ctx, 30*time.Second)` to make sure Talos doesn't hang forever

Reply:
in each usage, right?

Reply:
I thought about adding it here, as we make call to `fetchConfig`

In other words, Talos shouldn't hang forever on downloading the config or any other query to cloud metadata server